src/parser/schwabJSONEACHistoryParser.ts

```diff
@@ -39,30 +39,23 @@ const FIELD_LAPSE_AWARD_SHARES_DEPOSITED = 'NetSharesDeposited';
 const FIELD_LAPSE_AWARD_TOTAL_TAX = 'Taxes';
 
 // TODO: Add JSON parsing for options transactions
-/*
 // "","Award ID","Shares Exercised","Award Price","Sale Price","Award Type","Award Date",
```

Comment:
These kinds of rows were leftovers from the CSV parsing logic (showing an example of the CSV file header). They can be removed now that the code is doing JSON parsing.